client/src/main/java/com/tencent/rss/client/response/CompressedShuffleBlock.java

```diff
@@ -37,4 +39,8 @@ public int getUncompressLength() {
   public ByteBuffer getByteBuffer() {
     return byteBuffer;
   }
+
+  public long getBlockId() {
```

Comment:
remove.

Reply:
Without it, is it impossible to know which block is fetched from `readShuffleBlockData()`.
And this is helpful for debug.

Reply:
I don't think that we should add this field for debug.

Reply:
Removed.